config/environments/development.rb

```diff
@@ -37,5 +37,5 @@
 
   # config/environments/production.rb
 
-  Paperclip.options[:command_path] = "/usr/local/bin/"
+  Paperclip.options[:command_path] = "C:/Program Files/ImageMagick-6.9.1-Q16"
```

Comment:
Don't change this universally-- I would add `development.rb` to your `.gitignore` file. (If I push this it will no longer work for me)